frontend/src/metabase/public/constants.ts

```diff
@@ -4,5 +4,5 @@ export const DEFAULT_EMBED_DISPLAY_PARAMS = {
   titled: DEFAULT_DASHBOARD_DISPLAY_OPTIONS.titled,
   theme: undefined,
   hideParameters: DEFAULT_DASHBOARD_DISPLAY_OPTIONS.hideParameters,
-  downloadsEnabled: DEFAULT_DASHBOARD_DISPLAY_OPTIONS.downloadsEnabled,
+  downloadsEnabled: true,
```

Comment:
EmbedDisplayParams is an unparsed `string | boolean`, so we have to use `true` instead of `{ pdf: true, cardResult: true }` from the default display options.